lib/package/third_party/formatters/stylish.js

```diff
@@ -5,6 +5,7 @@
 "use strict";
 
 const chalk = require("chalk"),
+    stripAnsi = require("strip-ansi"),
```

Comment:
could you also update the `package.json` with this dependency?

Reply:
done